state-transition/core/interfaces.go

```diff
@@ -136,6 +136,6 @@ type ReadOnlyEth1Data interface {
 
 // ReadOnlyWithdrawals only has read access to withdrawal methods.
 type ReadOnlyWithdrawals interface {
-	EVMInflationWithdrawal() *engineprimitives.Withdrawal
+	EVMInflationWithdrawal(math.Slot) *engineprimitives.Withdrawal
```

Comment:
this interface was not duly updated when we made `EVMInflationWithdrawal` slot dependent